Services/UserService/Commands/EditUserCommand.cs

```diff
@@ -0,0 +1,35 @@
+﻿using DTO.RestRequests;
+using FluentValidation;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+using UserService.Interfaces;
+
+namespace UserService.Commands
+{
+    public class EditUserCommand : IEditUserCommand
+    {
+        private readonly IValidator<EditUserRequest> userInfoValidator;
+        private readonly IValidator<EditUserRequest> userPasswordValidator;
+        
+        public EditUserCommand
+            ([FromServices] IValidator<EditUserRequest> userInfoValidator, 
+            [FromServices] IValidator<EditUserRequest> userPasswordValidator)
+        {
+            this.userInfoValidator = userInfoValidator;
+            this.userPasswordValidator = userPasswordValidator;
+        }
+        
+        public Task<bool> Execute(EditUserRequest request)
+        {
+            if (!(string.IsNullOrEmpty(request.OldPassword)
+                    || string.IsNullOrEmpty(request.NewPassword)))
+            {
+                userPasswordValidator.ValidateAndThrow(request);
+            }
+            userInfoValidator.ValidateAndThrow(request);
+
+            throw new NotImplementedException();
```

Comment:
Why we throw exception here? May be change this on TODO?

Reply:
Yeah, You are right.

Reply:
I will add todo comment (but this file obviously will have another contributor that write all logic) 
but it returns Task , so I have to have NOtImplementedException here, if I don't want to write part of code of other persons?

Reply:
fixed